### PR TITLE
Utkarsha/91053/card view design change

### PR DESCRIPTION
**Pull request by utkarsha-deriv**

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -71,3 +70,27 @@ h6 {
   justify-content: center;
   align-items: center;
 }
+
+.card {
+  box-shadow: 0px 12px 16px -4px rgba(14, 14, 14, 0.08), 0px 4px 6px -2px rgba(14, 14, 14, 0.03) !important;
```
Comment (mohsen-deriv) [inline src/styles/index.scss:75]: please update your selector and don't use `!important`, thanks :pray:

File: src/styles/index.scss [hunk 2]
```diff
@@ -71,3 +70,27 @@ h6 {
...
+  border-radius: 16px;
+  color: rgba(65, 70, 82, 1);
+}
+
+[class*='cardTitle'] {
+  font-size: var(--ifm-h2-font-size) !important;
+  line-height: 2.25rem;
```
Comment (mohsen-deriv) [inline src/styles/index.scss:82]: use rem function

File: src/styles/index.scss [hunk 3]
```diff
@@ -71,3 +70,27 @@ h6 {
...
+}
+
+[class*='cardDescription'] {
+  margin-top: 8px !important;
+  line-height: 1.5rem;
```
Comment (mohsen-deriv) [inline src/styles/index.scss:87]: use rem function

File: src/styles/index.scss [hunk 4]
```diff
@@ -71,3 +70,27 @@ h6 {
...
+  font-size: var(--ifm-h4-font-size) !important;
+}
+
+[class*='cardContainer']::after {
+  content: url(/img/arrow_right_red.svg);
+  position: relative;
+  top: 1rem;
```
Comment (mohsen-deriv) [inline src/styles/index.scss:94]: use rem function

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: src/theme/DocCard/styles.module.css [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+.cardContainer:hover {
+  border-color: var(--ifm-color-primary);
+  box-shadow: 0 3px 6px 0 rgb(0 0 0 / 20%);
+}
+
+.cardContainer::after {
+  content: url(/img/arrow_right_red.svg);
+  position: relative;
+  top: 0.8rem;
```
Comment (sanjam-deriv) [inline src/theme/DocCard/styles.module.css:22]: use rem function

File: src/theme/DocCard/styles.module.css [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  border: 1px solid var(--ifm-color-emphasis-200);
+  border-radius: 16px;
+  color: rgba(65, 70, 82, 1);
+  transition: all var(--ifm-transition-fast) ease;
+  transition-property: border, box-shadow;
+}
+
+.cardContainer:hover {
+  border-color: var(--ifm-color-primary);
+  box-shadow: 0 3px 6px 0 rgb(0 0 0 / 20%);
```
Comment (sanjam-deriv) [inline src/theme/DocCard/styles.module.css:16]: Change to rem

File: src/theme/DocCard/styles.module.css [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  margin-bottom: 0;
+}
+
+.cardTitle {
+  font-size: var(--ifm-h2-font-size) !important;
+  line-height: 2.25rem;
+}
+
+.cardDescription {
+  margin-top: 8px;
```
Comment (sanjam-deriv) [inline src/theme/DocCard/styles.module.css:36]: use rem()  for sizing scss

**Review by sanjam-deriv — APPROVED** [review]

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**